src/pages/LoginRegister/RegisterPage.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useContext } from "react";
+import "./LoginPage.css";
+import AuthContext from "../../context/AuthProvider";
+import RegisterForm from "./RegisterForm";
+import { useNavigate } from "react-router-dom";
+
+export default function RegisterPage() {
+  const navigate = useNavigate();
+  const [errors] = useState("");
+  const { userDetails } = useContext(AuthContext);
+  /*testing array
+  const Users = [
+    {
+      email: "admin@123",
+      username: "admin",
+      password: "12ABqwer",
+    },
+    { email: "aa@123", username: "scsf", password: "12ABqwer" },
+  ];*/
+
+  const [isSuccess] = useState(false);
+
+  const Register = (details) => {
+    console.log(userDetails);
```

Comment:
Please remove the console.logs from the code, its generally not a good idea to have these on the frontend.
```suggestion

```

Reply:
okay, I'll delete that